build.sbt

```diff
@@ -15,7 +15,7 @@
 
 
 resolvers += Resolver.bintrayRepo("kamon-io", "snapshots")
-val kamonCore       = "io.kamon" %% "kamon-core"         % "1.1.3"
+val kamonCore       = "io.kamon" %% "kamon-core"         % "1.1.4-783ff3e53ccfae46c213b0cec8be48788d91fbcb"
```

Comment:
Yo @mladens, I just published kamon-core 1.1.4, please update this version and we are good to go.